dev/deps/spark-deps-hadoop-2.2

```diff
@@ -85,25 +89,41 @@ jackson-core-2.5.3.jar
 jackson-core-asl-1.9.13.jar
 jackson-databind-2.5.3.jar
 jackson-jaxrs-1.9.13.jar
+jackson-jaxrs-base-2.5.4.jar
+jackson-jaxrs-json-provider-2.5.4.jar
 jackson-mapper-asl-1.9.13.jar
+jackson-module-jaxb-annotations-2.5.4.jar
 jackson-module-scala_2.11-2.5.3.jar
 jackson-xc-1.9.13.jar
 janino-2.7.8.jar
+javassist-3.18.1-GA.jar
+javax.annotation-api-1.2.jar
 javax.inject-1.jar
+javax.inject-2.4.0-b31.jar
 javax.servlet-3.0.0.v201112011016.jar
 javax.servlet-3.1.jar
 javax.servlet-api-3.0.1.jar
+javax.ws.rs-api-2.0.1.jar
 javolution-5.5.1.jar
 jaxb-api-2.2.2.jar
 jaxb-impl-2.2.3-1.jar
 jcl-over-slf4j-1.7.10.jar
 jdo-api-3.0.1.jar
 jersey-client-1.9.jar
+jersey-client-2.22.1.jar
```

Comment:
The trouble with including both com.sun.jersey and org.glassfish.jersey is that you end up with multiple javax.ws.rs versions on the classpath - com.sun.jersey packages javax.ws.rs classes internally while org.glassfish.jersey requires providing the javax.ws.rs classes externally. This led to the discussion around SPARK-11081.